src/repositories/skill-category-repository.ts

```diff
@@ -6,3 +7,85 @@ export const getAllByFilters = async (where: Prisma.skill_categoriesWhereInput)
     where,
   })
 }
+
+export const getById = async (id: number) => {
+  return await prisma.skill_categories.findUnique({
+    where: {
+      id,
+    },
+  })
+}
+
+export const getByName = async (name: string) => {
+  return await prisma.skill_categories.findFirst({
+    where: {
+      name,
+    },
+  })
+}
+
+export const create = async (data: SkillCategory) => {
+  const currentDate = new Date()
+  const newSkillCategory = await prisma.skill_categories.findFirst({
+    orderBy: { sequence_no: "desc" },
+  })
+  const nextSequenceNo =
+    newSkillCategory?.sequence_no !== null && newSkillCategory?.sequence_no !== undefined
+      ? newSkillCategory.sequence_no + 1
+      : 1
+  const newCategory = await prisma.skill_categories.create({
+    data: {
+      ...data,
+      sequence_no: nextSequenceNo,
+      created_at: currentDate,
+      updated_at: currentDate,
+    },
+  })
+  return { message: "Successfully added", data: newCategory }
+}
+
+export const update = async (id: number, data: Prisma.skill_categoriesUpdateInput) => {
```

Comment:
Same for this function, move all business logic inside `skill-category-service` and only do prisma queries here

Reply:
done :3